MyConsoleApp/MyConsoleApp.Test/MyProgramTest.cs

```diff
@@ -7,11 +7,11 @@ public class MyProgramTest
         [Fact]
         public void VerifyGet()
```

Comment:
Change the test case name
also add more test cases

Reply:
made the necessary changes